doc/man/qvm.1

```diff
@@ -65,6 +65,9 @@ approximately <limit-ms> milliseconds. By default, this value is 0,
 which indicates an unlimited amount of time is allowed.
 .IP "--qubit-limit <n>"
 (Server Mode) Limit requests to consuming <n> qubits.
+.IP "--parallelization-limit <n>"
+Specify the number of qubits a program must contain before parallelization
+kicks in. The default value is unspecified.
```

Comment:
What happens if there is no value given? There *must* be a default no?

Reply:
There is a default but it's deliberately unspecified, since it might change on a per-machine basis in the future. The default is actually 19 currently.

Reply:
Hm. I think that's confusing, but also not.